R/extract.R

```diff
@@ -1,81 +1,82 @@
+#' Extract all conversation nodes as data frame.
+#' @param its_xml the xml tree of an .its file
 #' @export
-gather_conversations <- function(lena_log) {
+gather_conversations <- function(its_xml) {
   # Extract attributes from the conversation nodes
-  lena_log %>%
+  its_xml %>%
     xml_path_to_df(xpaths_bookmarks$conversation) %>%
     dplyr::mutate_(startTime = ~ convert_time_to_number(startTime),
                    endTime = ~ convert_time_to_number(endTime)) %>%
-    add_its_filename(lena_log)
+    add_its_filename(its_xml)
 }
 
+
+
+#' Extract all pause nodes as data frame.
+#' @param its_xml the xml tree of an .its file
 #' @export
-gather_pauses <- function(lena_log) {
-  lena_log %>%
-    xml_path_to_df("./ProcessingUnit/Recording/Pause") %>%
+gather_pauses <- function(its_xml) {
+  its_xml %>%
+    xml_path_to_df(xpaths_bookmarks$pause) %>%
     dplyr::mutate_(startTime = ~ convert_time_to_number(startTime),
                    endTime = ~ convert_time_to_number(endTime)) %>%
```

Comment:
Making an issue to convert these functions to use the new nonstandard evaluation code.